packages/hop-node/src/config/config.ts

```diff
@@ -33,13 +33,17 @@ export const minPriorityFeePerGas = normalizeEnvVarNumber(process.env.MIN_PRIORI
 export const priorityFeePerGasCap = normalizeEnvVarNumber(process.env.PRIORITY_FEE_PER_GAS_CAP)
 export const maxGasPriceGwei = normalizeEnvVarNumber(process.env.MAX_GAS_PRICE_GWEI)
 export const timeTilBoostMs = normalizeEnvVarNumber(process.env.TIME_TIL_BOOST_MS)
+export const awsAccessKeyId = process.env.AWS_ACCESS_KEY_ID
+export const awsSecretAccessKey = process.env.AWS_SECRET_ACCESS_KEY
+export const awsRegion = process.env.AWS_REGION
+export const awsProfile = process.env.AWS_PROFILE
 const envNetwork = process.env.NETWORK || Network.Kovan
 const isTestMode = !!process.env.TEST_MODE
 const bonderPrivateKey = process.env.BONDER_PRIVATE_KEY
 
-export const bondableChains: string[] = [Chain.Optimism, Chain.Arbitrum]
+export const oruChains: string[] = [Chain.Optimism, Chain.Arbitrum]
 export const rateLimitMaxRetries = 5
-export const rpcTimeoutSeconds = 10 * 60
+export const rpcTimeoutSeconds = 90
```

Comment:
It looks like this is the only place we use this variable is here. I think we should remove the `* 1000` in this line.

https://github.com/hop-protocol/hop/blob/develop/packages/hop-node/src/decorators/rateLimitRetry.ts#L28

Reply:
I think I am incorrect. I am going to resolve this.